src/sentry/search/eap/columns.py

```diff
@@ -0,0 +1,67 @@
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any, Literal
+
+from sentry_protos.snuba.v1.trace_item_attribute_pb2 import AttributeAggregation, AttributeKey
+
+from sentry.search.eap import constants
+
+
+@dataclass(frozen=True)
+class ResolvedColumn:
+    # The alias for this column
+    public_alias: str  # `p95() as foo` -> `foo` or `p95()` -> `p95()`
```

Comment:
The comment is a little ambiguous
```suggestion
    public_alias: str  # `p95() as foo` has the public alias `foo` and `p95()` has the public alias `p95()`
```